code/modules/holodeck/HolodeckControl.dm

```diff
@@ -63,6 +63,7 @@
 	"Meetinghall" 		= new/datum/holodeck_program(/area/holodeck/source_meetinghall),
 	"Courtroom" 		= new/datum/holodeck_program(/area/holodeck/source_courtroom, list('sound/music/traitor.ogg')),
 	"Chessboard"		= new/datum/holodeck_program(/area/holodeck/source_chess),
+	"Micro Building Area"		= new/datum/holodeck_program(/area/holodeck/source_smoleworld),
```

Comment:
```suggestion
	"Micro Building Area"		= new/datum/holodeck_program(/area/holodeck/source_smoleworld), //VOREStation add
```
Since this is a Polaris file, changes from the Polaris version should be marked with comments

Reply:
Uhh... Wait this... should be in a VR file? Will I need to send this bit up to polaris?